app/views/api/v1/accounts/articles/_article.json.jbuilder

```diff
@@ -0,0 +1,14 @@
+json.id article.id
+json.category_id article.category_id
+json.title article.title
+json.content article.content
+json.description article.description
+json.status article.status
+json.account_id article.account_id
+json.portal do
+  json.partial! 'api/v1/accounts/portals/portal.json.jbuilder', portal: article.portal
+end
+json.views article.views
+json.author do
```

Comment:
If the author is nil, would it break the API?

Reply:
Also missed from the last review, if the author is deleted does it nullify the article author? What is the current behavior?